File: app/src/main/java/com/talktiva/pilot/Talktiva.java
package com.talktiva.pilot;

import android.app.Application;
import android.content.res.Configuration;

public class Talktiva extends Application {

    public static String TAG;
    private static Talktiva talktiva;

    public Talktiva() {
        super();
    }

    public static Talktiva getInstance() {
        return talktiva;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        talktiva = this;
        TAG = getResources().getString(R.string.app_name);
    }

    @Override
    public void onLowMemory() {
        super.onLowMemory();
    }

    @Override
    public void onTrimMemory(int level) {
        super.onTrimMemory(level);
    }

    @Override
    public void onConfigurationChanged(Configuration newConfig) {
        super.onConfigurationChanged(newConfig);
    }

    @Override
    public void onTerminate() {
        super.onTerminate();
    }
}

File: app/src/main/java/com/talktiva/pilot/Talktiva.kt
package com.talktiva.pilot

import android.app.Application

class Talktiva : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
        TAG = resources.getString(R.string.app_name)
    }

    companion object {
        var TAG: String? = null
        var instance: Talktiva? = null
            private set
    }
}
